templates/default/src/ui/wallet/chain-switcher.tsx

```diff
@@ -1,16 +1,13 @@
 import { Button, ButtonGroup, FormControl, FormLabel, useToast } from "@chakra-ui/react";
-import {
-  mainnetChainsArray,
-  testnetChains,
-  useAccount,
-  useActiveChain,
-  useConnect,
-  useSuggestChainAndConnect,
-} from "graz";
+import { useAccount, useActiveChain, useConnect, useSuggestChainAndConnect } from "graz";
+import { getChainData, getChainDataArray } from "graz/chains";
 
 export const ChainSwitcher = () => {
   const toast = useToast();
 
+  const mainnetChainsArray = getChainDataArray(["cosmoshub", "juno", "sommelier", "stargaze", "osmosis"]);
+  const { osmosistestnet } = getChainData("osmosistestnet");
```

Comment:
Put outside component to prevent data refetching (beteween imports and component declaration)